### PR TITLE
docs(retries) document the retries setting

### PR DESCRIPTION
**Pull request by Tieske**
note: this also has to go to the 0.11 branch

### CODE REVIEW

File: app/docs/0.10.x/proxy.md [hunk 1]
```diff
@@ -598,7 +599,30 @@ More information on this topic is covered in the
 
 [Back to TOC](#table-of-contents)
 
-#### 4. Response
+#### 4. Errors & retries
+
+Whenever an error occurs during proxying Kong will use the underlying Nginx
```
Comment (theetrain) [inline app/docs/0.10.x/proxy.md]: Whenever an error occurs `while` proxying`,` Kong will use the underlying Nginx retries to pass the request `onto` the next upstream.

File: app/docs/0.10.x/proxy.md [hunk 2]
```diff
@@ -598,7 +599,30 @@ More information on this topic is covered in the
...
+   means an error or timeout occuring while establishing a connection with the
+   server, passing a request to it, or reading the response header.
+
+The second option is based on Nginx's directive
+[proxy_next_upstream][proxy_next_upstream]. This option is not directly
+configurable through Kong, but can be added using a custom Nginx configuration.
+See the [configuration reference][configuration-reference] for more details.
+
+
+[Back to TOC](#table-of-contents)
```
Comment (theetrain) [inline app/docs/0.10.x/proxy.md]: Back to `Table of Contents`

Comment (Tieske) [inline app/docs/0.10.x/proxy.md]: not changing this as this is used all over this document

File: app/docs/0.10.x/proxy.md (hunk 1 above)
Comment (Tieske) [inline app/docs/0.10.x/proxy.md]: updated

File: app/docs/0.10.x/proxy.md [hunk 3]
```diff
@@ -598,7 +599,31 @@ More information on this topic is covered in the
 
 [Back to TOC](#table-of-contents)
 
-#### 4. Response
+#### 4. Errors & retries
+
+Whenever an error occurs during while proxying, Kong will use the underlying
+Nginx [retries][ngx-http-proxy-retries] to pass the request onto the next
```
Comment (p0pr0ck5) [inline app/docs/0.10.x/proxy.md]: "underlying Nginx `mechanism`"

Comment (p0pr0ck5) [inline app/docs/0.10.x/proxy.md]: "on to", not "onto"

File: app/docs/0.10.x/proxy.md [hunk 4]
```diff
@@ -598,7 +599,31 @@ More information on this topic is covered in the
...
+upstream.
+
+There are two configurable elements here:
+
+1. The number of retries. This can be configured per API using the `retries`
```
Comment (p0pr0ck5) [inline app/docs/0.10.x/proxy.md]: "The number of retries." is a sentance fragment. consider refactoring this to be a complete sentence

File: app/docs/0.10.x/proxy.md [hunk 5]
```diff
@@ -598,7 +599,31 @@ More information on this topic is covered in the
...
+   property of the `API` object. See the [management API][API] for more
+   details on this.
+
+2. What exactly constitutes an error. Here Kong uses the Nginx defaults, which
+   means an error or timeout occuring while establishing a connection with the
+   server, passing a request to it, or reading the response header.
+
+The second option is based on Nginx's directive
+[proxy_next_upstream][proxy_next_upstream]. This option is not directly
```
Comment (p0pr0ck5) [inline app/docs/0.10.x/proxy.md]: "Nginx's proxy_next_upstream directive"

**Pull request merged**